draft-ietf-quic-transport.md

```diff
@@ -849,7 +849,10 @@ initial_stream_id_for_type) can be opened (see {{long-packet-types}}).  Initial
 limits are set in the transport parameters (see
 {{transport-parameter-definitions}}) and subsequently limits are advertised
 using MAX_STREAMS frames ({{frame-max-streams}}). Separate limits apply to
-unidirectional and bidirectional streams.
+unidirectional and bidirectional streams. When a MAX_STREAMS frame is received
+that permits more streams than can be expressed by the stream ID
+(i.e. greater than 2^60 - 1), the connection MUST be immediately closed with a
```

Comment:
I wonder if "greater than 2^60 - 1" is correct. 2^60 is a valid number to be carried by a MAX_STREAM frame.

To avoid confusion (between number of streams and the maximum stream IDs), it might be a good idea to simply state something like: _When a MAX_STREAM frame carrying Maximum Streams greater than 2^60 is received, the connection MUST be ..._

Reply:
Yeah, I think that is better.  Or "If a MAX_STREAMS carries a value of 2^60 or greater (which implies a maximum stream ID that cannot be expressed in the protocol), then the connection MUST ..."

Reply:
@martinthomson's text looks fine with one exception: s/a value of 2^60 or greater/a value greater than 2^60/.

2^60 is a valid number to be carried by MAX_STREAMS.

Reply:
Yep, my bad.  Fence-post error on my part.